guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>We can reword <code>When you initialize the AutoDL class it calculates</code> to <code>When AutoDL class is initialized, it calculates</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>